service/test/_job_client_test.js

```diff
@@ -190,7 +190,7 @@ describe('JobClient', function() {
 
       var client = new JobClient(fakeRestApiClient);
       var query = client.createQuery(null, null, fakePageSize);
-      query.next(function() {});
+      query.next(undefined, function() {});
```

Comment:
>(undefined, function() {}); [](start = 16, length = 27)

Why were these changed?

Reply:
Good catch. Looks like it was left by accident (doesn't change anything really though).